app/assets/javascripts/kaui/kaui.js

```diff
@@ -261,3 +268,57 @@ function isBlank(value) {
         return false;
     }
 }
+
+// this function set popover for all tags that have class object-id-popover
+// attributes:
+//      data-id = content of the popover,object id; required
+//      title = title of the popover; not required
+//      id = (must be {{id}}-popover) used to close popover when the copy image is clicked; if present; if not present a timeout of 5s will apply; not required
+function setObjectIdPopover(){
+    $(".object-id-popover").each(function(idx, e){
+        $(this).popover('destroy');
+        $(this).off("shown.bs.popover");
+
+        $(this).popover({
+            html: true,
+            content: function() {
+                var template = '<div class="{{id}}-content" >' +
+                    '{{id}}&emsp;<i id="{{id}}-copy" class="fa fa-clipboard" aria-hidden="true"></i> ' +
```

Comment:
Can we also change the cursor when highlight the clipboard (something like `{cursor: pointer;}`)?